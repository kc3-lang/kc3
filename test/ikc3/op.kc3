quote 1 + 20
1 + 20
quote 1 + 20 / 3
1 + 20 / 3
quote to_lisp(quote 1 + 20 / 3)
to_lisp(quote 1 + 20 / 3)
quote 1 + 20 / 3 * 4
1 + 20 / 3 * 4
quote to_lisp(quote 1 + 20 / 3 * 4)
to_lisp(quote 1 + 20 / 3 * 4)
quote 1 + 20 / 3 * 4 - 5
1 + 20 / 3 * 4 - 5
quote to_lisp(quote 1 + 20 / 3 * 4 - 5)
to_lisp(quote 1 + 20 / 3 * 4 - 5)
quote 20 / 3 * 4 - 5
20 / 3 * 4 - 5
quote to_lisp(quote 20 / 3 * 4 - 5)
to_lisp(quote 20 / 3 * 4 - 5)
quote (1 + 20)
(1 + 20)
quote to_lisp(quote (1 + 20))
to_lisp(quote (1 + 20))
quote a = (1 + 20)
a = (1 + 20)
