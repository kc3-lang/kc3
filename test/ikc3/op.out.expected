1 + 20
21
1 + 20 / 3
7
1 + 20 / 3 * 4
25
1 + 20 / 3 * 4 - 5
20
20 / 3 * 4 - 5
19
(1 + 20)
21
a = (1 + 20)
21
